data/translations/ja.ts

```diff
@@ -23,7 +23,7 @@
     </message>
     <message>
         <source>Warning, Caps Lock is ON!</source>
-        <translation>注意: Caps ロックがオンになっています!</translation>
+        <translation>注意: Caps Lock が有効です!</translation>
```

Comment:
I want to revert this translation except for ロック → Lock.
The reason why I reverted the most part:

- I quickly googled and *ON*/*OFF* (*オン*/*オフ*) looks like a bit more popular expression than *有効*/*無効* in Japanese. In addition, the original text is not *enabled* but *ON*, I think *オン* is a better translation than *有効*.
- I think になっています is better to express the current *status* of Caps Lock than です.

I also removed the exclamation mark (!) because it sounds a bit too emphasized in Japanese.

```suggestion
        <translation>注意: Caps Lock がオンになっています</translation>
```